src/Symfony/Component/VarDumper/Caster/FrameStub.php

```diff
@@ -18,15 +18,17 @@
  */
 class FrameStub extends EnumStub
 {
-    public $srcContext;
+    const DEFAULT_SRC_CONTEXT = 1;
```

Comment:
I still added it because is opens for some flexibility, even if it's global. The component already has some such public static for global configuration. For a dumping tool, that's fine IMO. Default values are overwritable when the default global config is not required.